docs/templates/explanation-template.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Noun or nouns describing your subject
+content_type: explanation
+no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+beta: true # Labels the page as beta; adds a banner to the top of the page.
+disable_image_expand: true # Stops images from expanding in a modal on click. Sets it for the entire page.
+class: no-copy-code # Disables the copy code button for all code blocks on the page.
+---
+
+<!-- Add an introduction paragraph here. Good introductions explain who this explanation topic is for and what this guide will cover. -->
+<!-- See https://documentation.divio.com/explanation/ for more info about how to write an explanation article -->
+<!-- See the following examples of explanation documentation:
+* https://docs.konghq.com/gateway/latest/key-concepts/routes/
+* https://docs.konghq.com/gateway/latest/production/deployment-topologies/hybrid-mode/
+* https://docs.konghq.com/gateway/latest/kong-enterprise/plugin-ordering/
+-->
+<!-- Remove these comments once you are done writing -->
+
+## First section <!-- Header optional if there's only one section in the article -->
+
+<!-- The first section should be about the main idea of the topic -->
+<!-- You can also use bulleted lists and paragraphs to describe -->
+
+Here's information about this topic. This feature has the following benefits:
+
+* First item
+* Second item
+
```

Comment:
```suggestion

### Subsection <!-- Optional --> 

 <!--  Go into more detail about some aspect of the functionality you're describing -->
 
 <!-- H2 is the highest header level a topic can have. If using nested headers, make sure that they're sequential. An H2 can contain an H3, but not the other way around. -->
 
```
Adding this to make it hopefully more obvious that you don't have to just use H2s + some guidance. It seems like common sense but you'd be surprised how many of these we've had to fix.